src/main/java/com/videoclub/suriken/security/model/MyUserDetail.java

```diff
@@ -0,0 +1,64 @@
+package com.videoclub.suriken.security.model;
+
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+public class MyUserDetail implements UserDetails {
+
+    private final String userName;
+    private final String password;
+    private final boolean active;
+    private final Set<SimpleGrantedAuthority> authorities = new HashSet<>();
+
+    public MyUserDetail(AppUser appUser) {
+        this.userName = appUser.getUserName();
+        this.password = appUser.getPassword();
+        this.active = true;
+
+        appUser.getRoles().forEach(role -> {
```

Comment:
Random pitanje. Da li si znas sta su java streams i kako se koriste? To je baas bitno za intrerview

Reply:
Znam. Proveo sam pre koji mesec dosta vremena ucenja o stream-ovima.. Znam da mogu da otvorim stream, pa da uradim forEach ili ovako kao sto sam ja bez srteam-a. Znaci dobija se isti rezultat, samo sto stream, ako ima mogucnosti primeni neke optimizacije, kao sto su paralelno izvrsavanje...